www/css/style.css

```diff
@@ -21,10 +21,6 @@ div#debuggingWindow {
     background: white;
 }
 
-.fh_appform_field_input_container .slider.slider-horizontal {
```

Comment:
Didn't expect this to be deleted. 

I think that https://github.com/feedhenry/Appforms-Template-v3/pull/23 may need a rebase and regeneration.